runner.py

```diff
@@ -54,6 +55,14 @@ def run_agent(terminate_event, callback_event, config_file, name, team, index, m
 
 
 def main():
+    # Inject DLL
+    injector_dir = os.path.join(os.path.dirname(__file__), "RLBot_Injector.exe")
+    incode=subprocess.call([injector_dir, 'hidden'])
+    injector_codes=['INJECTION_SUCCESSFUL','INJECTION_FAILED','MULTIPLE_ROCKET_LEAGUE_PROCESSES_FOUND','RLBOT_DLL_ALREADY_INJECTED','RLBOT_DLL_NOT_FOUND','MULTIPLE_RLBOT_DLL_FILES_FOUND']
+    print(injector_codes[incode])
+    if incode!=0 or incode!=3:
```

Comment:
I'm pretty sure this is equivalent to `if true`. You probably want `if not (incode == 0 or incode == 3)`

Reply:
yep